src/main.rs

```diff
@@ -155,6 +159,20 @@ fn main() {
             .set(ids.chimper, ui);
         }
 
+        const PAD: Scalar = 20.0;
```

Comment:
This is just copied from the example and has no meaning in this program. A proper variable name should be used instead.